pep517/build.py

```diff
@@ -1,15 +1,14 @@
 """Build a project using PEP 517 hooks.
 """
 import argparse
-import io
 import logging
 import os
 import shutil
+import tomli
```

Comment:
tomli is a third-party module, so it should go in a separate group, after the stdlib modules and before the modules that are part of this package. I'm aware I don't always follow this convention very strictly itself, but I think it's a good opportunity to follow it here.

(This can be cleaned up in a few other files as well - I won't comment on each one)

Reply:
:+1: Will fix. 

I recommend adopting a tool such as isort to do this step automatically. I think it will make contributing and reviewing simpler.

I've added isort in #137.